libs/sdk-ui-filters/src/AttributeFilter/AttributeDropdown/types.ts

```diff
@@ -1,8 +1,17 @@
-// (C) 2019-2020 GoodData Corporation
+// (C) 2019-2021 GoodData Corporation
 import { IPagedResource, IAttributeElement } from "@gooddata/sdk-backend-spi";
 
+/**
+ * @public
+ */
 export const emptyListItem = { empty: true };
```

Comment:
Can we avoid exporting `emptyListItem` publicly? Even if it means specifying `EmptyListItem` explicitly.